app/util.py

```diff
@@ -1,11 +1,26 @@
 from app import mongo
 from bson.objectid import ObjectId
+import requests
 
 
 def serialize_doc(doc):
     doc["_id"] = str(doc["_id"])
     return doc
 
+def slack_msg(msg):
+    webhook_url = 'https://hooks.slack.com/services/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'

```

Comment:
first, always always always. never every put again in future always config config confg

Reply:
i have told everytime. but for some reason this happens again and again